src/views/dashboard/Calendar.vue

```diff
@@ -142,16 +145,11 @@ export default {
         statutoryPromise,
         onlinePromise
       ]).then(([regular, statutory, online]) => {
-        this.events = [...regular, ...statutory, ...online]
+        this.calendarOptions.events = [...regular, ...statutory, ...online]
         this.isLoading = false
       })
     }
   },
-  mounted () {
```

Comment:
@HassanCehef won't it stop the events list from being loaded?

Reply:
He locally made an event and that was being loaded, so no

Reply:
The fetch function was called twice due to this "mounted".

Reply:
okay, if it works then this makes sense indeed